ta4j-core/src/test/java/org/ta4j/core/mocks/MockBarBuilder.java

```diff
@@ -23,18 +23,15 @@
  */
 package org.ta4j.core.mocks;
 
-import java.time.Clock;
 import java.time.Duration;
 import java.time.Instant;
-import java.time.ZoneOffset;
-
 import org.ta4j.core.bars.BaseBar;
 import org.ta4j.core.bars.BaseBarBuilder;
 import org.ta4j.core.num.NumFactory;
 
 public class MockBarBuilder extends BaseBarBuilder {
 
-    private Clock clock = Clock.fixed(Instant.ofEpochMilli(0), ZoneOffset.UTC);
+    private Instant beginTime = Instant.now();
```

Comment:
I would still use predefined Instant.EPOCH at least for debugging purpose. It is harder to debug code that have each run in different time span.

Example:
First run have bars 00:00, 00:01, 00:02. Second run in a nother part of day 12:30, 12:31, 12:32.
These differences may discract from real cause of indicator miscalculation (Renko case?).


From another point of view random time may prove it works in any case. Maybe it should be configurable by tester, what he want to test like `mockbarBilder.withStartAtEpoch()` or `mockbarBuilder.witnStartAtRealTime()`.

Reply:
Can I suggest creating a static method that would provide incremental end dates that would be used by both `MockBarBuilder` and `MockBarSeriesBuilder` keeping consistency between both?

Reply:
Yes, I have replaced now by EPOCH.

Reply:
Wouldn't be better if you pulled my branch to yours? This PR may stay.